src/vs/workbench/browser/parts/views/media/views.css

```
@@ -278,8 +278,8 @@
 	display: none;
 }
 
-.viewpane-filter > .viewpane-filter-controls > .monaco-action-bar .action-item .action-label.codicon.filter {
-	padding: 2px;
+.viewpane-filter > .viewpane-filter-controls > .monaco-toolbar .action-item .action-label.codicon-filter {
```

@sandy081, as far as I can tell, the old rule was never used because `monaco-action-bar` was not a direct child of `viewpane-filter-controls`.

---

You are right and this can be confirmed by other usages of `.monaco-action-bar`. I would consider following suggestion to be consistent with other rules

```suggestion
.viewpane-filter > .viewpane-filter-controls .monaco-action-bar .action-item .action-label.codicon-filter {
```

---

Actually I would delete this rule and change the following rule by adding side margins to the badge

https://github.com/microsoft/vscode/blob/467e1971fc471b5f6c82cf826ab1bcfb12c59cd5/src/vs/workbench/browser/parts/views/media/views.css#L270-L274

```css
.viewpane-filter-container > .viewpane-filter > .viewpane-filter-controls > .viewpane-filter-badge {
	margin: 4px 2px;
	padding: 0px 8px;
	border-radius: 2px;
}
```

---

Done.

---

Missed your newer comment, updating now.

---

Updated, but I kept the left margin on the badge `0px` since there's already a lot of space on that side.